azure/template.json

```diff
@@ -0,0 +1,211 @@
+        },
+        "aspSize": {
+            "type": "string",
+            "defaultValue": "1",
+            "metadata": {
+                "description": "The size of the App Service Plan."
+            }
+        },
+        "aspInstances": {
+            "type": "int",
+            "defaultValue": 1,
+            "metadata": {
+                "description": "The instance count of the App Service Plan."
+            }
+        },
+        "customHostName": {
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The DNS hostname for the service."
+            }
+        },
+        "loggingRedisConnectionString": {
+            "type": "string",
+            "metadata": {
+                "description": "The connection string for the Redis logging cache."
+            }
+        },
+        "keyVaultCertificateName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the certificate stored in the shared keyvault."
+            }
+        },
+        "keyVaultName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the shared keyvault."
+            }
+        },
+        "keyVaultResourceGroupName": {
+            "type": "string",
+            "metadata": {
+                "description": "The resource group name of the shared keyvault."
+            }
+        }
+    },
+    "variables": {
+        "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
+        "resourceNamePrefix": "[toLower(concat('das-', parameters('environmentName'),'-', parameters('serviceName')))]",
+        "appServiceName": "[concat(variables('resourceNamePrefix'), '-as')]",
+        "appServicePlanName": "[concat(variables('resourceNamePrefix'), '-asp')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2017-05-10",
+            "name": "app-service-plan",
+            "type": "Microsoft.Resources/deployments",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'),'app-service-plan.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "appServicePlanName": {
+                        "value": "[variables('appServicePlanName')]"
+                    },
+                    "aspSize": {
+                        "value": "[parameters('aspSize')]"
+                    },
+                    "aspInstances": {
+                        "value": "[parameters('aspInstances')]"
+                    }
+                }
+            }
+        },
+        {
+            "apiVersion": "2017-05-10",
+            "name": "app-service-certificate",
+            "resourceGroup": "[parameters('sharedResourceGroup')]",
```

Comment:
Why is this certificate going to the shared resource group?

Reply:
Removed the app-service-certificate deployment from being deployed to a shared resource group as there is a new app service plan being deployed specifically for this app service. If the app service were being deployed to a shared app service plan, then I could of used this method.